### PR TITLE
chore(rand_name): moves random name generator to its own pkg

### PR DESCRIPTION
**Pull request by bartoszmajsak**
and adds unit tests for it (with custom gomega matchers)

(ideally these should be property-based tests, but I will work on that when done with #56)

### CODE REVIEW

File: pkg/naming/name_generator_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	})
+
+	It("should always generate letter when single character name is requested", func() {
+		name := naming.RandName(1)
+		Expect(name).To(OnlyContain("abcdefghijklmnopqrstuvwxyz"))
+	})
+
+	It("should generate name with first character being a letter", func() {
+		name := naming.RandName(32)
+		Expect(name).To(StartWithLetter())
```
Comment (aslakknutsen) [inline pkg/naming/name_generator_test.go]: these are a bit random if they fail or not are they not?

Comment (bartoszmajsak) [reply pkg/naming/name_generator_test.go]: If there's a bug - yes. But there is not :)

As I said in the PR description - ideally it should be done using generators, but I would rather first ship that and then use generators to improve tests.

Comment (aslakknutsen) [reply pkg/naming/name_generator_test.go]: Why not just remove '-' from the options. It's not like it's required :)

Comment (bartoszmajsak) [reply pkg/naming/name_generator_test.go]: Still, cannot be a number

Comment (bartoszmajsak) [reply pkg/naming/name_generator_test.go]: We could simplify though and remove those tests, true. You favor this over what we have now?

Comment (aslakknutsen) [reply pkg/naming/name_generator_test.go]: we don't need to use numbers either.. :)

Comment (bartoszmajsak) [reply pkg/naming/name_generator_test.go]: ACK

Comment (bartoszmajsak) [reply pkg/naming/name_generator_test.go]: Fixed with 862840b. I just wanted it to look fancy :P

**Pull request merged**